docs/reference/cli/database.md

```diff
@@ -312,6 +312,47 @@ Request: "albums 0 5 artist_id:19<LF>"
 Response: "albums 0 5 artist_id:19 count:1 id:5 album:Bounce%20[Single]<LF>"
 ```
 
+***
+## works
+`works <start> <itemsPerResponse> <taggedParameters>`
+
+The `works` query returns all works known by the server.
+
+A Work is a (usually classical) piece of music made up of one or more movements, acts, scenes etc, each of which is usually a separate track.
+
+A Work is defined by its name and its composer.
+
+An Album can contain many Works, and an instance of a Work can exist on many Albums. There is therefore a many-to-many relationship between Works and Albums.
+This many-to-many relationship is resolved in the Tracks table which contains the album id and the work id.
+
+**Accepted tagged parameters:**
+
+| Tag | Description |
+|---|---|
+| `search` | Search string. The search is case insensitive and obeys the `Search Within Words` server parameter. |
+| `artist_id` | Limit results to those works on which the artist identified by `artist_id` performs. |
+| `genre_id` | Limit results to the genre(s) identified by `genre_id`. The genre_id may be a list of comma separated IDs. |
+| `library_id` | Virtual library ID, to restrict the results to a virtual library view. |
+| `work_id` | Limit results to an individual `work`. |
+| `role_id` | Limit results to the role(s) identified by `role_id`. The role_id may be a list of comma separated IDs. |
+
+**Returned parameters:**
+
+| Result | Tag | Description |

```

Comment:
Please use the same structure as for other queries (with the blocks, and the "for each work" etc.).

Reply:
Sorry, didn't quite understand what was going on with that markup - is it right now?

How to I look at how what I've created is rendered?